addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14266,13 +14276,13 @@ msgstr ""
 #. Description of setting "Videos -> Library -> Flatten TV show seasons" with label #20412
 #: system/settings/settings.xml
 msgctxt "#36144"
-msgid "Remove the TV show season node, toggles between 'If only one season' (default), 'Always' and 'Never'."
+msgid "Remove the TV show season node, toggles between \"If only one season\" (default), \"Always\" and \"Never\"."
```

Comment:
atm its

msgctxt "#36144"
msgid "Remove the TV show season node, toggles between [If only one season] (default), [Always] and [Never]."
msgstr ""

Please suggest a complete rewording then, the current format still applies as intead of spinner you get a list but still have to explain what they mean.